monkey/monkey_island/cc/ui/src/components/report-components/security/ScannedServers.js

```diff
@@ -32,17 +33,30 @@ class ScannedServersComponent extends React.Component {
   }
 
   render() {
+
     let defaultPageSize = this.props.data.length > pageSize ? pageSize : this.props.data.length;
     let showPagination = this.props.data.length > pageSize;
+
+    const howManyScannedMachines = this.props.data.length;
+    const reducerFromScannedServerToServicesAmount = (accumulated, scannedServer) => accumulated + scannedServer["services"].length;
+    const howManyScannedServices = this.props.data.reduce(reducerFromScannedServerToServicesAmount, 0);
```

Comment:
`, 0` also not necessary.

Reply:
Considered best practice to avoid unexpected + behaviour, like concating strings or stuff